providers/base/tests/test_wifi_client_test_netplan.py

```diff
@@ -157,3 +205,468 @@ def test_parser_mutually_exclusive_fail(self):
         ):
             with self.assertRaises(SystemExit):
                 parse_args()
+
+    @patch(
+        "sys.argv",
+        [
+            "script.py",
+            "-i",
+            "wlan0",
+            "-s",
+            "SSID",
+            "-d",
+            "--renderer",
+            "networkd",
+        ],
+    )
```

Comment:
Minor: I like it better with the arg patch in the test and not as a decorator. It's a bit easier to follow. or even:
```
args = ["script.py", "-i", "wlan0", "-s", "SSID", "-d", "--renderer", "networked"]
with patch("sys.argv", args):
    ...
```

Reply:
Fixed~~